clients/gitlabrepo/contributors.go

```diff
@@ -84,7 +84,34 @@ func (handler *contributorsHandler) retrieveUsers(queryName string) ([]*gitlab.U
 	return users, nil
 }
 
+// Orginisation is an experimental feature in GitLab. Takes a slice
+// pointer to maintain list of domains and email returns an anonymised
+// string with the slice index as a unique identifier.
+// Pointer is required as we are modifying the slice length.
+func getOrginisation(orgDomain *[]string, orgReal string) string {
```

Comment:
Should be in `clients/gitlabrepo/commits.go` since we're dealing with commit email addresses.

Reply:
I did not put that there since this should only be a temporary measure and this is specific to GitLab if you want me to move it after happy to do so.